src/ng/http.js

```diff
@@ -11,6 +11,8 @@ var JSON_PROTECTION_PREFIX = /^\)\]\}',?\n/;
 
 function paramSerializerFactory(jQueryMode) {
 
+  return jQueryMode ? jQueryLikeParamSerializer : paramSerializer;
```

Comment:
If we are going to do this then we might as wel ditch the `paramSerializerFactory` altogether and just reference these two serializers directly in their respective providers.